package.json

```diff
@@ -52,14 +53,15 @@
   },
   "lint-staged": {
     "*.+(js|jsx)": [
-      "eslint --fix --max-warnings=0",
-      "prettier --write"
+      "npm run eslint:fix",
+      "prettier --write \"**/*.+(js|jsx)\""
     ],
-    "*.+(json|yml|yaml|md|vue)": [
-      "prettier --write"
+    "*.+(json|yml|yaml|md)": [
+      "prettier --write \"**/*.+(json|yml|yaml|md)\""
     ],
     "*.+(css|scss)": [
-      "sass-lint-auto-fix"
+      "npm run scss-lint:fix",
+      "npm run scss-lint"
```

Comment:
Do we need to run `scss-lint` again after running `scll-lint:fix`? Would not it throw error if it could not fix something?

Reply:
Exactly that is what we want, errors like class-name-format and so many others cannot be fixed automatically. For those cases, we would want to prevent a commit until a manual fix is done.